src/Options/index.js

```diff
@@ -46,7 +46,9 @@ export interface ParseServerOptions {
   :ENV: PARSE_SERVER_APPLICATION_ID */
   appId: string;
   /* Your Parse Master Key */
-  masterKey: string;
+  masterKey: (() => void) | string;
+  /* MasterKeyTtl is the duration in seconds for which the master key is refreshed. Only valid if the masterKey is a function. */
+  masterKeyTtl: ?number;
```

Comment:
```suggestion
  masterKey: (() => void) | string;
  /* (Optional) The duration in seconds for which the current `masterKey` is being used before it is requested again if `masterKey` is set to a function. If `masterKey` is not set to a function, this option has no effect. Default is `0`, which means the master key is requested by invoking the  `masterKey` function every time the master key is used internally by Parse Server. */
  masterKeyTtl: ?number;
```

Should it also set a default with `:DEFAULT: 0`? May make things easier as the value can't be undefined.